api/app/signals/settings/base.py

```diff
@@ -478,7 +478,7 @@
 
 # SIG-1017
 FEEDBACK_ENV_FE_MAPPING = {
-    'LOCAL': 'http://dummy_link',
+    'LOCAL': os.getenv('FEEDBACK_ENV_FE_MAPPING', 'http://localhost:3001'),
```

Comment:
for local development we have the `development.py` settings file where we can override these settings

Reply:
For me this seemed to be most in line with the approach with `ACCEPTANCE` and `PRODUCTION` below.